### PR TITLE
Improve Ledger API versioning docs

### PR DESCRIPTION
**Pull request by bame-da**
CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by nemanja-da — APPROVED** [review]
ty

**Comment by kamil-da** [comment]
@bame-da Are you sure that the docs are compiled properly with this change? The link to the `version service` doesn't seem to work when I compile the docs.

File: docs/source/support/compatibility.rst [hunk 1]
```diff
@@ -25,7 +25,11 @@ Application Portability and to some extent Network Upgradeability are achieved b
 
 Specifically, if a DAML Application is built against Ledger API version X.Y.Z and a Participant Node exposes Ledger API version X.Y2.Z2, the application is guaranteed to work as long as Y2.Z2 >= Y.Z.
 
-Before SDK 1.7, the Ledger API version exposed by the Participant Node matches the SDK version. Currently, these versions can diverge. Integration Components, DAML Drivers, and Participant Nodes advertise the Ledger API version they expose.
+Participant Nodes advertise the Ledger API version they support via the :ref:`version service <version-service>`˛.
```
Comment (kamil-da) [inline docs/source/support/compatibility.rst:28]:
```suggestion
Participant Nodes advertise the Ledger API version they support via the :ref:`version service <version-service>`.
```
I think there's an invalid character before the dot.

**Comment by bame-da** [comment]
@kamil-da that's odd. I copied it from https://github.com/digital-asset/daml/blob/9a2317cbdbd7b9bcbdf5b9a012beb3e6e567a921/docs/source/app-dev/ledger-api.rst#L44.

**Comment by kamil-da** [comment]
It works fine now. Thank you :)

**Review by kamil-da — APPROVED** [review]
LGTM

**Pull request merged**